src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -1130,6 +1130,24 @@ public void CheckAndMarkAsPageable(DottedNameNode node)
             }
         }
 
+        public void CheckAndMarkAsDelegatable(FirstNameNode node)
+        {
+            Contracts.AssertValue(node);
+            Contracts.AssertIndex(node.Id, _typeMap.Length);
+
+            var info = GetInfo(node).VerifyValue();
+            if (info.Kind == BindKind.PowerFxResolvedObject && SupportsPaging(node))
```

Comment:
>BindKind.PowerFxResolvedObject [](http://example.com/codeflow?start=29&length=30)

This bindkind is used for basically every resolved global symbol in PowerFx. Combined with the checks in supportsPaging/ArgValidator, should be fine, but might be worth adding a clarifying comment about that.